srcjs/bindings/messageBarInputBinding.js

```diff
@@ -1,10 +1,13 @@
 // Input binding
+import { getAppInstance } from "../init.js";
+
 var f7MessageBarBinding = new Shiny.InputBinding();
 
 $.extend(f7MessageBarBinding, {
 
   initialize: function(el) {
-    app.messagebar.create({
+    this.app = getAppInstance();
+    this.app.messagebar.create({
```

Comment:
@hypebright: if we stored the instance into `this.instance =this.app.messagebar.create(...)`, we would not need to do all the `this.app.messagebar.get`. Not super fan about `instance` word. Maybe something shorter, except that `el` is already taken.

Reply:
`inst`, `object`, `obj`?